pyls/plugins/jedi_completion.py

```diff
@@ -51,16 +51,18 @@
 @hookimpl
 def pyls_completions(config, document, position):
     try:
-        definitions = document.jedi_script(position).completions()
+        code_position = _utils.position_to_jedi_linecolumn(document, position)
```

Comment:
Is this something that could fail?

If not we could move it out of the try?

Reply:
Should be moved out, since the catch is only there for missing CompiledObject attributes.

Reply:
The `try...catch` can be removed. Fixed upstream https://github.com/davidhalter/jedi/issues/1452